internal/restconfig/restconfig.go

```diff
@@ -213,6 +205,13 @@ func (rcp *Producer) RunOnSelectedContext(function PerContextFn, status reporter
 		return status.Error(err, "error building the cluster.Info for the default configuration")
 	}
 
+	if clusterInfo.Submariner != nil {
```

Comment:
We should also check `ServiceDiscovery` i case connectivity isn't installed.

```suggestion
        var version string
	if clusterInfo.Submariner != nil {
	    version = clusterInfo.Submariner.Spec.Version
	} else if clusterInfo.ServiceDiscovery != nil {
	    version = clusterInfo.ServiceDiscovery.Spec.Version
	}
	
	if version != "" {
```